src/blockdigest.cpp

```diff
@@ -16,8 +16,7 @@ std::vector<uint8_t> base_filter::outpoint_data(const COutPoint& outpoint) const
 {
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << outpoint;
-    std::vector<uint8_t> data(stream.begin(), stream.end());
-    return data;
+    return std::vector<uint8_t>(stream.begin(), stream.end());
```

Comment:
It's still better to use the other -- then you can reduce it down to 1 prevector, so all your data is on the stack 

e.g.:

```
prevector<32, uint8_t> x;
outpoint_data(x, outpoint);
contains(x).
```